physics/cu_c3_driver.F90

```diff
@@ -279,6 +280,14 @@ subroutine cu_c3_driver_run(ntracer,garea,im,km,dt,flag_init,flag_restart,&
 !$acc end kernels
      endif
 
+
+     if(progsigma)then
```

Comment:
Should this piece be in the _init() phase? And only done once?

Reply:
yes, good idea